### ISSUE TITLE
Cannot Install Code

### ISSUE THREAD

**Issue by ohbus** [id1945/ngx-scanner-qrcode#6]
code ran `ng add ngx-scanner-qrcode`

```bash
git:(master) ng add ngx-scanner-qrcode               
ℹ Using package manager: npm
⚠ Unable to find compatible package. Using 'latest' tag.
⚠ Package has unmet peer dependencies. Adding the package may not succeed.

The package ngx-scanner-qrcode@1.0.25 will be installed and executed.
Would you like to proceed? Yes
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: lunch-booking-frontend@0.0.0
npm ERR! Found: @angular/common@15.0.0
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^15.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^8.0.3" from ngx-scanner-qrcode@1.0.25
npm ERR! node_modules/ngx-scanner-qrcode
npm ERR!   ngx-scanner-qrcode@"1.0.25" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/subho/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/subho/.npm/_logs/2022-11-18T22_50_11_534Z-debug-0.log
✖ Packages installation failed, see above.
```

*** 

```bash
Angular CLI: 15.0.0
Node: 16.15.0
Package Manager: npm 8.5.5
OS: darwin arm64

Angular: 15.0.0
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, material, platform-browser, platform-browser-dynamic
... router, service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1500.0
@angular-devkit/build-angular   15.0.0
@angular-devkit/core            15.0.0
@angular-devkit/schematics      15.0.0
@schematics/angular             15.0.0
rxjs                            7.5.7
typescript                      4.8.4
```

***

[eresolve-report.txt](https://github.com/id1945/ngx-scanner-qrcode/files/10045358/eresolve-report.txt)
[2022-11-18T22_50_11_534Z-debug-0.log](https://github.com/id1945/ngx-scanner-qrcode/files/10045359/2022-11-18T22_50_11_534Z-debug-0.log)

**Comment by id1945** [comment]
I will check it

**Comment by ohbus** [comment]
Thanks,
Subhrodip.

**Comment by id1945** [comment]
I published Version 1.0.26 for Angular 15. 
👍 Now you can use it.
